src/generators/samples/rlcSamples.ts.hbs

```diff
@@ -1,28 +1,47 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-import createClient, {
-  {{clientClassName}}, 
-} from "{{clientPackageName}}";
-{{#if hasCredentials }}
-import { DefaultAzureCredential } from "@azure/identity";
-{{/if}}
+{{#each importedTypes}}
+{{this}}
+{{/each}}
 import * as dotenv from "dotenv";
 
 dotenv.config();
 
+{{#each samples}}
 /**
- * This is a simple sample on how to create client and please add your own code below
+ * This sample demonstrates how to {{this.description}}
  *
+ * @summary {{this.description}}
+ * x-ms-original-file: {{this.originalFileLocation}}
  */
-async function main() {
+async function {{name}}() {
   {{#each this.clientParamAssignments}}
   {{this}}
   {{/each}}
-  const client: {{clientClassName}} = createClient({{this.clientParameterNames}});
-  const result = !!client ? "Success" : "Fail";
-  console.log(result," to create client.");
-  // Add your own code here
+  const client = {{this.defaultFactoryName}}({{this.clientParamNames}});
+  {{#each this.pathParamAssignments}}
+  {{this}}
+  {{/each}}
+  {{#each this.methodParamAssignments}}
+  {{this}}
+  {{/each}}
+  {{#if this.isPaging}}
+  const initialResponse = await client.path({{this.pathParamNames}}).{{method}}({{methodParamNames}});
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  {{else if this.isLRO}}
```

Comment:
what if a method is both LRO and Paginate ?

Reply:
Currently it will go to the first matched condition `this.isPaging` if both LRO and Paginate. Another way to handle this is to ignore this case in transformation because we don't support this now. Which one do you prefer?